scriptTest.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+if [ "$1" = "install" ]; then
+	clear
+	echo "=========== Begin to install and build ========="
+	sudo python3.4 setup.py install
+	sudo python3.4 setup.py build
+	echo "=========== Installed and built the sniffles ========="
+	echo
+elif [ "$1" = "test" ]; then
+	clear
+	echo "=========== Begin to install and build ========="
+	sudo python3.4 setup.py install
```

Comment:
Isn't the purpose of `scriptTest test` running unit tests? If so, it does not require installing files, which needs the root privilege. `runtest.py` uses files in the build directory only.